### DISCUSSION TITLE
TW地区搜索出现问题了

### DISCUSSION THREAD

**Discussion by ccplusone** [ywmoyue/biliuwp-lite#44] [Q&A] [answered]
各种各样的报错都有 大陆和港澳的都正常 服务器也换了很多个 想大佬帮忙解决一下

**Comment by modebra** [comment]
港澳用的是哪个代理？

**Reply by ccplusone** [reply]
默认的服务器 https://example.com/ 首选代理是https://b.chuchai.vip

**Comment by ywmoyue** [comment]
请参考这篇内容将日志文件发送到我邮箱 https://github.com/ywmoyue/biliuwp-lite/discussions/45

**Reply by ccplusone** [reply]
已发送 谢谢

**Reply by ywmoyue** [reply]
好的，已收到

**Reply by ywmoyue** [reply]
根据你日志中显示用于请求台湾地区搜索的代理地址，我这边简单测了下：
| 请求失败 | 500错误 | 账号未登录错误 | 成功 |
| :---: | :---: | :---: | :---: |
| 泰区评论会寄.lushx.top | api.mahiron.moe | 陈睿.我爱你 | 哔哩.plus |
|  | 伊蕾娜.我爱你 | bilibili.com.co | api.qiu.moe |
|  |  |  | atri.ink |
|  |  |  | hk.xifeng.cf |

其中请求失败可能是服务器关了或其他原因导致连不上，500错误表示服务器内部错误，账号未登录错误我不清楚什么原因，但由于确实有请求成功的搜索，我认为app中是没问题的

**Reply by ywmoyue** [reply, marked as answer]
如果安卓端使用出问题的代理也无法进行搜索的话，就可以去哔哩漫游那边反馈

**Reply by ccplusone** [reply]
我这边试了一下 提示解析服务器缺少cookie 然后我朋友也说没有用

**Comment by ywmoyue** [comment]
麻烦测试一下漫游搜索和播放是否可用 https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.4.3
